pkgs/applications/science/biology/prodigal/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "prodigal";
+  version = "2.6.3";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    repo = "Prodigal";
+    owner = "hyattpd";
+    rev = "v${version}";
+    sha256 = "1fs1hqk83qjbjhrvhw6ni75zakx5ki1ayy3v6wwkn3xvahc9hi5s";
+  };
+
+  installPhase = ''
```

Comment:
This can be replaced with `makeFlags`:
```diff
diff --git c/pkgs/applications/science/biology/prodigal/default.nix i/pkgs/applications/science/biology/prodigal/default.nix
index 214a386e750..d5ae402879a 100644
--- c/pkgs/applications/science/biology/prodigal/default.nix
+++ i/pkgs/applications/science/biology/prodigal/default.nix
@@ -12,11 +12,8 @@ stdenv.mkDerivation rec {
     sha256 = "1fs1hqk83qjbjhrvhw6ni75zakx5ki1ayy3v6wwkn3xvahc9hi5s";
   };
 
-  installPhase = ''
-    mkdir -p $out/bin
-    cp prodigal $out/bin
-  '';
-  
+  makeFlags = [ "CC=cc" "INSTALLDIR=$(out)/bin" ];
+
   meta = with stdenv.lib; {
     description = "Fast, reliable protein-coding gene prediction for prokaryotic genomes";
     homepage = https://github.com/hyattpd/Prodigal;
```